### DISCUSSION TITLE
Email to all Wertgutschein receiver

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretix/pretix#2630] [Q&A]
Hi,

2020 after canceling the event our guest had three options:
- refund the whole ticket (order status "storniert")
- transform it into a Wertgutschein for the next Event (order status "storniert")
- transform it into a Wertgutschein for the next Event but donate a part of this. (order status "storniert (bezahlte Gebühr)")
   - and there is min. one order with complete refund and 0€ donation but status "storniert (bezahlte Gebühr)" i don´t really know how this could happen.

Now we want to send a E-Mail to all People with a Wertgutschein to explain how they can use it. (and a few things more).

Is ther any way to do this? Also with pretix-batch-emailer i don´t find a way fot this :/

Thank you very much! :)

**Comment by raphaelm** [comment]
No, I don't think we have a good way to do it. You could try getting the email address via the "payments and refunds" export and then mailing outside the system.

**Reply by navlisilvan** [reply]
ah. i wanted to use the Placeholder of the ticketsystem to link their Wertgutschein.
Is there a option the other way around, when i find out all oder number to batch-mail there list like the pretix-batch-emailer does?
